app/src/main/java/org/scesi/cappuchino/ui/screens/homeScreen.kt

```diff
@@ -86,38 +86,33 @@ fun HomeScreenContent(title: String) {
         )
         Spacer(modifier = Modifier.height(dimensionResource(id = R.dimen.margin_padding_size_medium)))
         SearchBar()
-        Spacer(modifier = Modifier.height(dimensionResource(id = R.dimen.margin_padding_size_medium)))
 
-        Column(
-            modifier = Modifier
-                .fillMaxSize()
-                .verticalScroll(rememberScrollState())
-        ) {
-            TextBoxCarreras(text = "Licenciatura en Ing en Sistemas", onClick = {})
-            TextBoxCarreras(text = "Licenciatura en Ing en Informática", onClick = {})
-            TextBoxCarreras(text = "Licenciatura en Ing en Industrial", onClick = {})
-            TextBoxCarreras(text = "Licenciatura en Ing en Electrónica", onClick = {})
-            TextBoxCarreras(text = "Licenciatura en Ing en Sistemas", onClick = {})
-            TextBoxCarreras(text = "Licenciatura en Ing en Informática", onClick = {})
-            TextBoxCarreras(text = "Licenciatura en Ing en Industrial", onClick = {})
-            TextBoxCarreras(text = "Licenciatura en Ing en Electrónica", onClick = {})
-            TextBoxCarreras(text = "Licenciatura en Ing en Sistemas", onClick = {})
-            TextBoxCarreras(text = "Licenciatura en Ing en Informática", onClick = {})
-            TextBoxCarreras(text = "Licenciatura en Ing en Industrial", onClick = {})
-            TextBoxCarreras(text = "Licenciatura en Ing en Electrónica", onClick = {})
-            TextBoxCarreras(text = "LICENCIATURA EN BIOLOGÍA", onClick = {})
-            TextBoxCarreras(text = "Licenciatura en Ing en Informática", onClick = {})
-            TextBoxCarreras(text = "Licenciatura en Ing en Industrial", onClick = {})
-            TextBoxCarreras(text = "Licenciatura en Ing en Electrónica", onClick = {})
-        }
 
     }
 }
 
 @OptIn(ExperimentalMaterial3Api::class)
 @Composable
 fun SearchBar() {
```

Comment:
move this composable to another file 
ui -> utils(new package) -> SearchBar.kt

Reply:
Also try to pass a list of searchable items , 
right now the search bar only works for "carreras" , you can create an interface , and every type of list we want in the future will implement the interface , and we can pass it into the SearchBar composable 
something like 
fun SearchBar(searchList: List<SearchableItem>) {
.....